2.cellprofiler_processing/README.md

```diff
@@ -8,16 +8,11 @@ To run segmentation, we use the standard approach used in CellProfiler for Cell
 
 This standard approach is as follows:
 
-- **IdentifyPrimaryObjects:** Identify nuclei from the `d0` channel images. This creates a group for an object called `OrigNuclei`, which includes nuclei that are a part of cells that touch an edge of the image.
+- **IdentifyPrimaryObjects:** Identify nuclei from the `d0` channel images (discarding ones that touch the edges of the image). This creates a group for an object called `Nuclei`.
```

Comment:
Consider using one sentence per-line here.

Reply:
I can not use one sentence per line here since I am using the hashes/bullet points. If I move the one sentence, it creates a new hash for the line. I need to keep all sentences for each module in their own individual hash.